### ISSUE TITLE
Classes break BEM convention

### ISSUE THREAD

**Issue by certainlyakey** [undergroundwires/ez-consent#15]
It is mentioned in the README that internal CSS classes follow [the BEM convention](https://en.bem.info). However, in one aspect the convention is broken - some classes contain an element inside of an element, such as [a "read more" class or "OK" class](https://github.com/undergroundwires/ez-consent/blob/26b9c52411b669c011873d8667fa655d0b3f6b67/src/ez-consent.js#L26): 

```
const cssClassNames = {
      root: 'cookie-consent',
      hide: 'cookie-consent__hide',
      buttons: {
        shared: 'cookie-consent__buttons-button',
        read_more: 'cookie-consent__buttons__read-more',
        ok: 'cookie-consent__buttons__close',
      },
};
```

According to BEM [this should not happen](https://en.bem.info/methodology/faq/#why-not-create-elements-of-elements-block__elem1__elem2).

**Comment by undergroundwires** [comment]
Hi @certainlyakey,
Thank you for the input. We can fix it in non-breaking way. What names would you suggest instead?

**Comment by undergroundwires** [comment]
Since [0.13.0](https://github.com/undergroundwires/ez-consent/releases/tag/1.3.0) I have added ability to customize CSS names [as documented here](https://github.com/undergroundwires/ez-consent?tab=readme-ov-file#2-initialize-the-script) which may be helpful, would still change the names if I get suggestion from someone who knows about this more than me.

**Comment by undergroundwires** [comment]
Renamed:

- `cookie-consent__hide`: `cookie-consent--hidden`
- `cookie-consent__buttons-button`: `cookie-consent__button`
- `cookie-consent__buttons__read-more`: `cookie-consent__button--more`
- `cookie-consent__buttons__close`: `cookie-consent__button--ok`

Old classes are preserved for backwards compatibility.

**Comment by undergroundwires** [comment]
I just released [v1.3.0](https://github.com/undergroundwires/ez-consent/releases/tag/1.3.0) with ability to choose custom CSS names and fix to BEM naming in backwards compatible way.
